tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -1,13 +1,73 @@
 """Set permissions for users."""
 from collections import ChainMap
+from copy import deepcopy
+from typing import List, Optional, TYPE_CHECKING
 
+from django.apps import apps
+from django.db.models import Q, QuerySet
 from rest_framework import permissions
 
+if TYPE_CHECKING:  # pragma: no cover
+    from django.contrib.auth.models import Permission  # pragma: no cover
```

Comment:
We don't use this import, but importing it this was allows us to use it in type hints without actually importing it in production code.